modules/openapi-generator/src/test/java/org/openapitools/codegen/python/PythonClientExperimentalTest.java

```diff
@@ -270,7 +270,7 @@ public void arrayModelTest() {
         Assert.assertEquals(cm.classname, "sample.Sample");
         Assert.assertEquals(cm.classVarName, "sample");
         Assert.assertEquals(cm.description, "an array model");
-        Assert.assertEquals(cm.vars.size(), 0);
+        Assert.assertEquals(cm.vars.size(), 1);  // there is one value for Childer definition
```

Comment:
What is the dataType for this sample?
It having vars of length 1 shows that my older hack is being used and that we have hoisted model information into a new variable.
How about we keep the model as is an use the information from
CodegenModel.dataType or CodegenModel.arrayModelType?
We can see all of the properties that are available by looking at that class [here](https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenModel.java#L29)

We can debug the CodegenModels and see the information in them by running the generate command with the `-DdebugModels=true` as [shown here](https://github.com/openapitools/openapi-generator/wiki/FAQ#how-to-debug-openapi-generator)

How about in the classvars template here
https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/python/python-experimental/model_templates/classvars.mustache#L115
Doing a check to see if the model is an arrayModel
and setting the datatype accordingly?
It would look like:
```
{{#isArrayModel}}
            'value': ({{{dataType}}},),  # noqa: E501
{{/isArrayModel}}
{{^isArrayModel}}
{{#requiredVars}}
            '{{name}}': ({{{dataType}}},),  # noqa: E501
{{/requiredVars}}
{{#optionalVars}}
            '{{name}}': ({{{dataType}}},),  # noqa: E501
{{/optionalVars}}
{{/isArrayModel}}
```
We would also need to do a similar change in 
[method_init_shared](https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/python/python-experimental/model_templates/method_init_shared.mustache#L2)
To:
- not loop over required an optional args if we have an array model, if we are that case, just set value as a required positional arg
- else use our existing looping over variables
- in the docstring if we are an arraymodel set value as a required positional arg

@jirikuncar 
What do you think about making these changes?
If you don't want to make them, we can keep using the existing code/(my older solution) where we shovel the model data into a value variable.

Reply:
I would prefer to keep the current code. It looks like too many changes and `{{#isArrayModel}}` guards are required for templates.

Reply:
Ah, understood. That's fine.